### PR TITLE
Feature/frontend websocket

### PR DESCRIPTION
**Pull request by VL4DYSL4V**

### CODE REVIEW

File: slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/presentationsession/PresentationSessionController.java [hunk 1]
```diff
@@ -27,7 +28,7 @@ public PresentationSessionController(PresentationSessionService presentationSess
 	@PostMapping("/new")
 	public ResponseEntity<CreateSessionResponseDto> createSession(
 			@RequestBody CreateSessionRequestDto createSessionRequestDto,
-			@RequestParam(name = "leaseDuration", defaultValue = "180") int leaseDuration) {
+			@RequestParam(name = "leaseDuration", defaultValue = "120") int leaseDuration) {
```
Comment (Anatolii-Ivanov) [inline slidez-core/src/main/java/com/binarystudio/academy/slidez/domain/presentationsession/PresentationSessionController.java]: Have you agreed this change with Kiril?

File: slidez-fe-core/src/common/routes/AuthenticationRoute.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ import { useAppSelector } from '../../hooks'
 import { selectIsLoggedIn } from '../../containers/user/store'
 
 // @ts-ignore
-const PublicRoute = ({ component: Component, ...rest }) => {
+const AuthenticationRoute = ({ component: Component, ...rest }) => {
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/routes/AuthenticationRoute.tsx]: I'd rather have Public and Private route

File: slidez-fe-core/src/services/ws/ws-endpoint.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+const WsEndpoint = Object.freeze({
+    ENDPOINT: process.env.REACT_APP_WEB_SOCKET_ENDPOINT || '',
+    SNAPSHOT_TOPIC: process.env.REACT_APP_WEB_SOCKET_SNAPSHOT_TOPIC || '',
+    CREATED_POLL_TOPIC:
+        process.env.REACT_APP_WEB_SOCKET_CREATED_POLL_TOPIC || '',
+    ANSWERED_POLL_TOPIC:
+        process.env.REACT_APP_WEB_SOCKET_ANSWERED_POLL_TOPIC || '',
+    SNAPSHOT_QUEUE: process.env.REACT_APP_WEB_SOCKET_SNAPSHOT_QUEUE || '',
+    CREATE_POLL_QUEUE: process.env.REACT_APP_WEB_SOCKET_CREATE_POLL_QUEUE || '',
+    ANSWER_POLL_QUEUE: process.env.REACT_APP_WEB_SOCKET_ANSWER_POLL_QUEUE || '',
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/services/ws/ws-endpoint.ts]: If I understand them correctly, write these variables statically, they won't change from server to server

File: slidez-fe-core/src/services/ws/ws-service.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+import { Message } from 'webstomp-client'
+import { SnapshotDto } from '../../containers/presentation_session/dto/SnapshotDto'
+
+export const helloConnect = () => {
+    const msg = { name: 'Alex' }
+    const onMessage = (message: Message) => {
+        alert(message.body)
+    }
+    const onConnectionSuccess = () => WsHelper.send('/slidez/hello', msg)
+    WsHelper.disconnect()
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/services/ws/ws-service.ts]: Why you make disconnect here?

File: slidez-fe-core/src/services/ws/ws-helper.ts [hunk 5]
```diff
@@ -0,0 +1,50 @@
+import SockJS from 'sockjs-client'
+import Stomp, { Client, Message } from 'webstomp-client'
+
+let socket: WebSocket | undefined = undefined
+let stompClient: Client | undefined = undefined
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/services/ws/ws-helper.ts]: Please don't create global variables...
You can use [singleton ](https://stackoverflow.com/questions/1479319/simplest-cleanest-way-to-implement-a-singleton-in-javascript)here

**Review by Anatolii-Ivanov — CHANGES_REQUESTED** [review]

**Review by Kosichenko — APPROVED** [review]

**Review by Anatolii-Ivanov — APPROVED** [review]

**Pull request merged**